src/Equinox/Tracing.fs

```diff
@@ -0,0 +1,107 @@
+module Equinox.Core.Tracing
+
+open System.Diagnostics
+
+let source = new ActivitySource("Equinox")
+
+module Tags =
+    // General tags
+
+    /// The full stream name
+    [<Literal>]
+    let stream_name = "eqx.stream_name"
+    /// The ID of the stream
+    [<Literal>]
+    let stream_id = "eqx.stream_id"
+    /// The category of the stream
+    [<Literal>]
+    let category = "eqx.category"
+    /// The store being used
+    [<Literal>]
+    let store = "eqx.store"
+
+    // Information about loading events
+
+    /// The version of the stream at read time
+    [<Literal>]
+    let read_version = "eqx.read_version"
+    /// The configured batch size
+    [<Literal>]
+    let batch_size = "eqx.batch_size"
+    /// The total number of batches loaded from the store
+    [<Literal>]
+    let batches = "eqx.batches"
+    /// The total number of events loaded from the store
+    [<Literal>]
+    let loaded_count = "eqx.count"
+    /// The total bytes loaded during Transact/Query
```

Comment:
```suggestion
    /// Total size of events loaded
```
(alluding to fact it includes storage overhead; should ideally refer to 'raw' or something to allude to fact that it would be the compressed size of the blob in the DDB store)